spec/rspec/core/example_spec.rb

```diff
@@ -58,6 +58,21 @@ def metadata_hash(*args)
     end
   end
 
+  it "captures example timing even for exceptions unhandled by RSpec" do
+    begin
+      avoided = RSpec::Support::AllExceptionsExceptOnesWeMustNotRescue::AVOID_RESCUING
+      unhandled = Class.new(StandardError)
+      avoided << unhandled
```

Comment:
Interestingly ruby 1.8.7 and jruby `rescue Exception` because of

```ruby
      if RSpec::Support::Ruby.jruby? || RUBY_VERSION.to_f < 1.9
        # :nocov:
        # For some reason, rescuing `Support::AllExceptionsExceptOnesWeMustNotRescue`
        # in place of `Exception` above can cause the exit status to be the wrong
        # thing. I have no idea why. See:
        # https://github.com/rspec/rspec-core/pull/2063#discussion_r38284978
        # @private
        AllExceptionsExcludingDangerousOnesOnRubiesThatAllowIt = Exception
        # :nocov:
      else
        # @private
        AllExceptionsExcludingDangerousOnesOnRubiesThatAllowIt = Support::AllExceptionsExceptOnesWeMustNotRescue
      end
```

(in `Example#run`):
```ruby
              rescue AllExceptionsExcludingDangerousOnesOnRubiesThatAllowIt => e
                set_exception(e)
```

https://github.com/rspec/rspec-core/pull/2063#discussion_r38284978 is super weird (head scratch). I guess I don't need to verify the rescue gets called here. I'm surprised that this behavior is apparently similar in jruby also. ah well.